packages/components/src/types/props/link-callbacks.ts

```diff
@@ -0,0 +1,25 @@
+import { Generic } from '@a11y-ui/core';
+import { Events } from '../../enums/events';
+import { EventValueOrEventCallback } from '../callbacks';
+
+/* types */
+export type LinkCallbacksPropType = {
+	[Events.onClick]?: EventValueOrEventCallback<Event, string>;
```

Comment:
Beim Link, gibt es kein Value im Callback. Das würde sich zum Button unterscheiden.

Reply:
Link ist gerade so implementiert, dass es den href als Value übergibt: `this.state._on?.onClick(event, this.state._href);`